src/main/java/org/myrobotlab/service/Arduino.java

```diff
@@ -2241,12 +2235,36 @@ public ServiceConfig getConfig() {
   @Override
   public ServiceConfig load(ServiceConfig c) {
     ArduinoConfig config = (ArduinoConfig) c;
+    
+    if (config.serial != null) {
+      serial = (Serial)Runtime.start(config.serial);
+    }
 
     if (config.port != null) {
       connect(config.port);
     }
     return c;
   }
+  
+  static public LinkedHashMap<String, ServiceConfig> getDefault(String name) {

```

Comment:
Where is the @ Override annotation..  the method signature is wrong.. it should include the type...  as per the interface.